try {  // in this code run if eveything smooth work perfect
  let x = y + 10; 
} catch (err) { // in this code run if any error in try section
  console.log(err.message); 
}
